portfolio/src/main/webapp/script.js

```diff
@@ -20,20 +20,27 @@ function loadPage() {
 
 //TO DO Change the += format to use Node.appendChild()
 function getComments(){
-  fetch('/data').then(response => response.json()).then((messages) => {
+  var quantity = document.getElementById('commentCount');
```

Comment:
Can it be `const`?

Reply:
No, since this fixes the amount, once selected.